easybuild/easyconfigs/n/npstat/npstat.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'MakeCp'
+
+name = 'npstat'
+version = '0.99'
+
+homepage = 'https://code.google.com/archive/p/npstat/'
+description = """npstat implements some population genetics tests and estimators that can be applied
+ to pooled sequences from Next Generation Sequencing experiments."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
```

Comment:
hmm, I may have to go with `GCC/4.9.3-2.25` here, because `npstat` needs to be combined with another tool (PR coming up) that requires OCaml (and OCaml doesn't build with `intel`, cfr. caml.inria.fr/mantis/view.php?id=7000)